docs/questions/query-builder/expressions/cumulativecount.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Cumulative count
+---
+
+# Cumulative count
+
+Cumulative count computes the additive total count of rows across a breakout.
+
+## Syntax
+
+You can use the pre-defined "Cumulative count" metric, or a custom expression:
+
+```
+CumulativeCount
+```
+
+Cumulative count can only be used in the query builder's **Summarize** step — you cannot use cumulative count to create a custom column or in a custom filter.
+
+## How cumulative count is computed
+
+Under the hood, cumulative count is computed in three steps:
+
+1. Group records using the breakout from the **Group by** block,
+2. Count the rows in each group,
+3. For each group, compute the cumulative total of the counts in all previous groups up to the current group.
+
+Because cumulative count uses values from previous rows, the sort order in the breakout column matters.
+
+For example, if you have a table with data grouped by month, cumulative count will be computed like this:
+
+| Month    | `Count` | `CumulativeCount` |
+| -------- | ------- | ----------------- |
+| July     | 5       | 5                 |
+| November | 4       | 5+4 = 9           |
+| March    | 2       | 5 + 4 + 2 = 11    |
+
+If the sort is changed (while values remain the same), then the cumulative count changes as well:
+
+| Month    | `Count` | `CumulativeCount` |
+| -------- | ------- | ----------------- |
+| March    | 2       | 2                 |
+| July     | 5       | 2 + 5 = 7         |
+| November | 4       | 2 + 5 + 4 = 11    |
+
+You can preview and check the results of your summary by clicking on the ▶ button to the right of the **Summarize** block.
+
+## Cumulative count with multiple breakouts
+
+Because `CumulativeCount` uses other rows, the order of data is important. When there are multiple breakout columns specified in **Group by**, there are multiple potential sort orders, so in Metabase, cumulative count will handle it as follows:
```

Comment:
```suggestion
Because `CumulativeCount` calculates its count based on previous rows, Metabase needs to determine what those previous rows are. It will always order the rows based on the first breakout. So if there are multiple breakout columns specified in **Group by**, Metabase will:
```